api/v1beta1/cinder_types.go

```diff
@@ -78,7 +169,7 @@ type Cinder struct {
 	Status CinderStatus `json:"status,omitempty"`
 }
 
-// +kubebuilder:object:root=true
+//+kubebuilder:object:root=true
```

Comment:
we should also have an IsReady on the overall state of cinder services specified by the CR.

https://github.com/openstack-k8s-operators/keystone-operator/blob/master/api/v1beta1/keystoneapi_types.go#L210

Reply:
Done